sample-apps/reviewer/client.py

```
@@ -0,0 +1,416 @@
+            if level:
+                review_info["level"] = level
+            if comment:
+                review_info["comment"] = comment
+            if reviewer_email:
+                review_info["reviewer_email"] = reviewer_email
+            if reviewer_name:
+                review_info["reviewer"] = reviewer_name
+
+            payload = {"info": json.dumps(review_info)}
+
+            response = requests.put(
```

Ruff (0.15.21)

[error] Probable use of `requests` call without timeout

(S113)